lib/rubocop/cop/style/string_hash_keys.rb

```diff
@@ -30,6 +41,14 @@ def autocorrect(node)
             corrector.replace(node.source_range, symbol_content)
           end
         end
+
+        private
+
+        def environments?(node)
```

Comment:
Is there a more descriptive method name that can be used?

Reply:
I removed the method because `on_pair` can directly call `receive_environments_method?` by the refactoring.